docs/pages/use_cases/COMPSs.md

```diff
@@ -0,0 +1,62 @@
+---
+page_id: COMPSs
+title: COMPSs Workflow Provenance
+description: COMPSs recording of Workflow Provenance for governance and reproducibility of computational experiments
+url: https://compss.bsc.es/
+image: COMPSs.png
+domains: [informatics] # should match item(s) in pages/domains
+tasks: [conduct_research, curation, data_handling, software_development] # should match item(s) in pages/tasks
+roles: [information_architect, researcher, software_developer] # should match item(s) in pages/roles
+---
+<!--
+   Copyright 2019-2022 RO-Crate contributors
+   <https://github.com/ResearchObject/ro-crate/graphs/contributors>
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+
+# COMPSs
+
+[![COMPSs logo](/assets/img/COMPSs.png)](https://compss.bsc.es/)
+
+[COMPSs](https://compss.bsc.es/) COMP Superscalar (COMPSs) is a task-based programming model which aims to ease the development of applications for distributed infrastructures, such as large High-Performance Clusters (HPC), Clouds and Container managed clusters. PyCOMPSs is the Python binding of COMPSs.
+
+COMPSs provides a programming interface for the development of the applications in Python/Java/C/C++, a runtime system that exploits the inherent parallelism of applications at execution time, and a rich ecosystem for the operation monitoring, performance evaluation and integration with Jupyter/Jupyterlab.
+
+The COMPSs runtime includes the capacity of automatically recording details of the application’s execution as metadata, also known as [Workflow Provenance](https://compss-doc.readthedocs.io/en/stable/Sections/05_Tools/04_Workflow_Provenance.html). The metadata is recorded in RO-Crate format, following [Workflow](https://w3id.org/workflowhub/workflow-ro-crate/1.0) and [Workflow Run Crate](https://w3id.org/ro/wfrun/workflow/0.4) profiles. With workflow provenance, you are able to share not only your workflow application (i.e. the source code) but also your workflow run (i.e. the datasets used as inputs, and the outputs generated as results). 
```

Comment:
```suggestion
The COMPSs runtime includes the capacity of automatically recording details of the application’s execution as metadata, also known as [Workflow Provenance](https://compss-doc.readthedocs.io/en/stable/Sections/05_Tools/04_Workflow_Provenance.html). The metadata is recorded in RO-Crate format, following [Workflow Crate](https://w3id.org/workflowhub/workflow-ro-crate/1.0) and [Workflow Run Crate](https://w3id.org/ro/wfrun/workflow/0.4) profiles. With workflow provenance, you are able to share not only your workflow application (i.e. the source code) but also your workflow run (i.e. the datasets used as inputs, and the outputs generated as results). 
```

Reply:
Maybe "Workflow RO-Crate" then? According to: https://about.workflowhub.eu/Workflow-RO-Crate/